app/components/elements/FontStyles.js

```diff
@@ -205,9 +205,8 @@ export const TopNavLink = Styled(NavLink)`
 
 export const TextLink = Styled(Link)`
```

Comment:
I think we should remove the `TextLink` styled-component here, as it's not being used, and it may confuse us down the road (which component to use for links)

Reply:
makes sense -- especially following making the best decisions for web rather than deferring to the prototype (as we talked about when starting components)